### DISCUSSION TITLE
Scroll Reveal Animation in React using Framer Motion

### DISCUSSION THREAD

**Discussion by giscus** [Evavic44/victoreke.com#99] [Announcements]
# Scroll Reveal Animation in React using Framer Motion

Discover the art of crafting captivating scroll reveal animations in your Reactjs application using the powerful, intuitive React Framer Motion library

https://victoreke.com/blog/scroll-reveal-animation-in-react-using-framer-motion

**Comment by fazi-gondal** [comment]
### Nice Work
